.doc_gen/metadata/transcribe_metadata.yaml

```diff
@@ -262,7 +262,20 @@ transcribe_StartStreamTranscriptionAsync:
               snippet_tags:
                 - transcribe.cpp.stream_transcription_async.code
   services:
-    transcribe: {StartStreamTranscriptionAsync}
+    transcribe: {StartStreamTranscription}
+transcribe_StartMedicalStreamTranscription:
```

Comment:
It looks like both of these APIs are part of a separate service client [transcribe_streaming](https://docs.aws.amazon.com/transcribe/latest/APIReference/API_streaming_StartMedicalStreamTranscription.html), so you'll need to create an entry for that in services.yaml and reference it here for the API ref links to work correctly.